.github/workflows/_deploy_production.yml

```diff
@@ -0,0 +1,100 @@
+name: Deploy Production
```

Comment:
I guess would be good to have `_deploy_staing.yml` too.

Reply:
Hm yeah we could, but I may do that in a separate PR. We only call it from one place, `cd.yml` so that's why I inlined it here.